src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -300,6 +300,16 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
 
         internal static async Task HandleDefinitionRequest(TextDocumentPosition textDocumentPosition, RequestContext<Location[]> requestContext)
         {
+            // Send a notification to signal that definition is sent
+            await requestContext.SendEvent(TelemetryNotification.Type, new TelemetryParams()
```

Comment:
Can we kick off telemetry & events in separate tasks? We don't need to await these as it can slow down overall processing